ui/src/pages/server-types/server-types.tsx

```diff
@@ -0,0 +1,117 @@
+// server-types.tsx
+import React, { useState } from 'react';
+import RadioButton from '../../components/radio-button/radio-button';
+
+type ServerType = {
+  id: string;
+  name: string;
+  subtext: string;
+};
+
+const serverTypes: ServerType[] = [
+  {
+    id: '1',
+    name: 'Small Instance',
+    subtext: 'Stable Environment with 2 CPU / 8GB RAM',
+  },
+  {
+    id: '2',
+    name: 'Small Instance',
+    subtext: 'Stable Environment with 2 CPU / 8GB RAM',
+  },
+  {
+    id: '3',
+    name: 'Medium Instance',
+    subtext: 'Stable Environment with 4 CPU / 16GB RAM',
+  },
+  {
+    id: '4',
+    name: 'Medium Instance',
+    subtext: 'Stable Environment with 4 CPU / 16GB RAM',
+  },
+  {
+    id: '5',
+    name: 'Large Instance',
+    subtext: 'Stable Environment with 6 CPU / 24GB RAM',
+  },
+  {
+    id: '6',
+    name: 'Large Instance',
+    subtext: 'Stable Environment with 6 CPU / 32GB RAM',
+  },
+  {
+    id: '7',
+    name: 'Large Instance',
+    subtext: 'Stable Environment with 8 CPU / 40GB RAM',
+  },
+  {
+    id: '8',
```

Comment:
This component however should be loading the server types from the `/spawner-profiles/` endpoint.

Reply:
These keys should be updated to match the endpoint as well. Here is the current mocked data: https://github.com/nebari-dev/jhub-apps/blob/f3ce17cf3e092476e94e33e2de509d6184ec4593/ui/src/data/api.ts#L17.

Please add descriptions to the existing items. And might also be good to add some additional items like you have to the dummy data to illustrate a larger list of server types